src/components/create/Create.js

```diff
@@ -42,6 +42,9 @@ const Create = ({ userID }) => {
   if (loading) return 'Submitting...';
   if (error) return `Submission error! ${error.message}`;
 
+  const options = breeds.map(breed=> ({label: breed.name, value: breed.name}))
```

Comment:
does the returned object need to be in parenthesis? Also, add a space in front of arrow function.

Reply:
Ill fix this!